src/server/routes/backend-wallet/import.ts

```diff
@@ -1,29 +1,48 @@
-import { Static, Type } from "@sinclair/typebox";
-import { FastifyInstance } from "fastify";
+import { Type, type Static } from "@sinclair/typebox";
+import type { FastifyInstance } from "fastify";
 import { StatusCodes } from "http-status-codes";
-import { WalletType } from "../../../schema/wallet";
 import { getConfig } from "../../../utils/cache/getConfig";
 import { createCustomError } from "../../middleware/error";
 import { AddressSchema } from "../../schemas/address";
 import { standardResponseSchema } from "../../schemas/sharedApiSchemas";
+import { getGcpKmsResourcePath } from "../../utils/wallets/gcpKmsResourcePath";
 import { importAwsKmsWallet } from "../../utils/wallets/importAwsKmsWallet";
 import { importGcpKmsWallet } from "../../utils/wallets/importGcpKmsWallet";
 import { importLocalWallet } from "../../utils/wallets/importLocalWallet";
 
 const RequestBodySchema = Type.Union([
   Type.Object({
-    awsKmsKeyId: Type.String({
-      description: "AWS KMS key ID",
-      examples: ["12345678-1234-1234-1234-123456789012"],
-    }),
+    label: Type.Optional(
```

Comment:
Since they all have label, it might be simpler to move the label to the top level
```typescript
schema = {
  label?: string
} & (
  | { <local wallet config> }
  | { <aws-kms wallet config> }
  | { <gcp-kms wallet config> }
)
```

That might look like this:
```typescript
const schema = Type.Intersect([
  Type.Object({
    label: Type.Optional(Type.String()),
  }),
  Type.Union([
    Type.Object({
      privateKey: Type.String(),
      // ...
    }),
    Type.Object({
      awsArn: Type.String(),
      // ...
    }),
    Type.Object({
      gcpKmsKeyRingId: Type.String(),
      // ...
    })
  ])
])
```